content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -580,9 +582,11 @@ Each feature gate is designed for enabling/disabling a specific feature:
   OpenTelemetry trace spans.
   See [Traces for Kubernetes System Components](/docs/concepts/cluster-administration/system-traces) for more details.
 - `LegacyServiceAccountTokenNoAutoGeneration`: Stop auto-generation of Secret-based
-  [service account tokens](/docs/reference/access-authn-authz/authentication/#service-account-tokens).
+  [service account tokens](/docs/concepts/security/service-accounts/#get-a-token).
+- `LegacyServiceAccountTokenCleanUp`: Clean up Secret-based
+  [service account tokens](/docs/concepts/security/service-accounts/#get-a-token).
```

Comment:
What does this "clean up" means to end users?
Is this really a user aware implementation change?

Reply:
> What does this "clean up" means to end users? 
> Is this really a user aware implementation change?

When this feature is default to be enabled in beta, those automatically generated service account token will be deleted if they are not been used for a time period (default to be one year). Please refer to 
[KEP]: https://github.com/kubernetes/enhancements/tree/master/keps/sig-auth/2799-reduction-of-secret-based-service-account-token for the rationale and https://github.com/kubernetes/kubernetes/pull/115554#event-9352349820 for the implementation.

Reply:
Thanks for the clarification. I believe this clarification could be incorporated into the feature description, or else people like me may treat it as an implementation clean up rather than token clean up.

Reply:
btw, as someone who is too bandwidth limited to check the design or the source code, I'd suggest we articulate user perceivable changes clearly.

Reply:
Thank you, Qiming. I updated the feature description.